flutter-studio/src/io/flutter/profiler/FlutterStudioMonitorStageView.java

```diff
@@ -724,6 +715,10 @@ public void selectionCleared() {
     final JBPanel legendPanel = new JBPanel(new BorderLayout());
     legendPanel.setOpaque(false);
     legendPanel.add(legendComponent, BorderLayout.EAST);
+    // Give the right-side a little space so it doesn't intersect with the right-side label.
```

Comment:
I assume you mean "get too close" not literally "intersect". Correct?

Reply:
Right.

Reply:
It's at the right-most edge w/o any gap, 30 pixels right border seems ok.

Reply:
I'll change comment to say that.